simple_ehm-runnable.py

```diff
@@ -124,7 +125,20 @@ def generate_tdata(ss, to, count, label):
 
 def generate_cut(ss, to, count):
     out_name = str(count) + video_path[-4:]
-    cuts.append(["ffmpeg", "-hide_banner", "-loglevel", "error", "-ss", ss ,"-i", video_path,  "-to", to, "-copyts"])
+    
+    tmp = datetime.datetime.strptime(ss, '%H:%M:%S.%f')
+    seconds = (tmp.hour * 60 + tmp.minute) * 60 + tmp.second
+
+    if (seconds > 10):
+		## calculate a safe place for non accurate point 5 seconds before, then it takes an accurate point for ss. For speed porpouse
+        safe = tmp - datetime.timedelta(seconds=5)
+        safe = safe.strftime("%H:%M:%S.%f")[:-3]
+        cuts.append(["ffmpeg", "-hide_banner", "-loglevel", "error", "-ss", safe, "-i", video_path, "-ss", ss, "-to", to, "-copyts"])
+
+    else:
+        cuts.append(["ffmpeg", "-hide_banner", "-loglevel", "error","-i", video_path,  "-ss", ss , "-to", to, "-copyts"])
+
+
```

Comment:
Can you further explain what problem does this solve and why does it work? I'm not getting what's the logic behind it. (anche in italiano se preferisci)
Also, this should be on a separate PR as it's unrelated to the original PR subject. Can you revert the changes related to this feature and open a separate PR for this? 
Thank you!

Reply:
Questo è un'ottimo risultato!
Grazie per la spiegazione.
Ricordati di copincollare questo blocco come oggetto del messaggio quando farai la PR per questa funzionalità!
È importante per mantenere ordinato lo storico e se ci sarà qualche problema in futuro, sarà sufficiente tornare al commit che introduce questo codice e leggere la descrizione per capire il come e il perché